run_clang_format.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find . -name '*.h' -or -name '*.hpp' -or -name '*.cpp' | xargs clang-format-3.8 -i -style=file $1
```

Comment:
Shouldn't this be 
```suggestion
find . -name '*.h' -or -name '*.hpp' -or -name '*.cpp' | xargs clang-format-14 -i -style=file $1
```

Reply:
with pre-commit, this file can be removed